automated/linux/ml-bench/pyarmnn-bench.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+
+usage() {
+    echo "Usage: $0 [-s <true|false>]" 1>&2
+    exit 1
+}
+
+while getopts "s:l:m:a:b:c:d:e:f:" o; do
+  case "$o" in
+    s) SKIP_INSTALL="${OPTARG}" ;;
+    a) LARMNN_TARBALL="${OPTARG}" ;;
+    d) TEST_DIR="${OPTARG}" ;;
+    *) usage ;;
+  esac
+done
+
+! check_root && error_msg "You need to be root to run this script."
```

Comment:
is this check needed for you to create an output dir?
We already do this check in function 'install_deps()'

Reply:
You're right I don't think this check is necessary, thanks for catching this